File: Q3.cpp
//  day3 coding Statement: Write a program to find ASCII values of a character
#include<iostream>
using namespace std;

int main()
{
    char ch;
    cout<<"Enter a character whose ascii value you seek: ";
    cin>>ch;

    cout<<"The Ascii value of "<< ch << " is: " << int(ch)<<"\n";
    return 0;

}

File: Q3.py
#Day 3 coding Statement: Write a program to find ASCII values of a character


char = input("Enter a character whose Ascii value you seek: ")

ascii_value = ord(char)        # ord function is used to get the ascii value for any character in python.

print(f"The Ascii value of this character is: {ascii_value}")